rclpy/test/test_utilities.py

```diff
@@ -18,6 +18,27 @@
 import rclpy.utilities
 
 
+class TestValidateRemoveRosArgs(unittest.TestCase):
+
+    def test_remove_ros_args(self):
+        args = ['process_name', '-d', '__ns:=/foo/bar', '__ns:=/fiz/buz', '--foo=bar', '--baz']
+        stripped_args = rclpy.utilities.remove_ros_args(args=args)
+        self.assertEqual(4, len(stripped_args))
+        self.assertEqual('process_name', stripped_args[0])
+        self.assertEqual('-d', stripped_args[1])
+        self.assertEqual('--foo=bar', stripped_args[2])
+        self.assertEqual('--baz', stripped_args[3])
+
+    def test_remove_ros_args_empty(self):
```

Comment:
Nit: It seems that this test can be simplified by replacing the try/except block with something like:
```
args = []
with self.assertRaises(RuntimeError):
    rclpy.utilities.remove_ros_args(args=args)
```

Reply:
I was just typing this :)

lgtm otherwise.

Reply:
Today I learned.